app/scripts/services/quota.js

```diff
@@ -277,6 +282,45 @@ angular.module("openshiftConsole")
         return _.some(quotas, isExceeded) || _.some(clusterQuotas, isExceeded);
     };
 
+    //Specific to storage quotas for max claim count and max storage usage
+    // and the amount of claims and storage already used
+
+    var getStorageQuotas = function(quotas) {
+      // Currently, there can be multiple quotas per namespace (bug?)
+      // So let's go through them and find the most restrictive
+      var maxNumberClaims = 0;
```

Comment:
the logic in here doesn't seem right to me, how is it handling if none of the quotas have storage restrictions, shouldn't these values be null and not 0 in that case.